librosa/core/intervals.py

```diff
@@ -464,13 +469,15 @@ def plimit_intervals(
         new_point = frontier.pop(best_f)
         intervals.append(new_point)
 
-        for seed in seeds:
-            new_seed = tuple(np.array(new_point) + np.array(seed))
+        for _ in seeds:
+            new_seed = tuple(np.array(new_point) + np.array(_))
```

Comment:
mypy got cranky about reusing the seed variable name here, so it's been swapped to _.